ReplacingDocker/podman.sh

```diff
@@ -0,0 +1,287 @@
+buildah_image() {
```

Comment:
For convenience, it might be nice to have a function that detects if podman is installed, and if not, then query the user about installing it and doing so if they say yes.